features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/BitmapPoolTest.kt

```diff
@@ -358,6 +361,53 @@ internal class BitmapPoolTest {
         assertThat(captor.firstValue).isEqualTo(0)
     }
 
+    @RepeatedTest(3000)
```

Comment:
let's reduce the number of repetitions to just few, probably 3000 runs takes too long and we can accumulate them in different runs on CI anyway.

Reply:
It takes about 4 seconds for 3000 runs (locally). I'll reduce it to 30